lib/wallet/txdb.js

```diff
@@ -827,9 +827,20 @@ class TXDB {
 
       const nameHash = covenant.getHash(0);
       const key = layout.o.encode(nameHash);
+      const hash = await this.bucket.get(key);
 
-      if (await this.bucket.has(key))
-        return true;
+      // Allow a double open if previous auction period has expired
+      // this is not a complete check for name availability or status!
+      if (hash) {
+        const names = this.wdb.network.names;
+        const period = names.biddingPeriod + names.revealPeriod;
+        const oldTX = await this.getTX(hash);
```

Comment:
Nit: remove brackets for 1 line if statement, `oldTX` -> `tx` variable naming for minimal style

Reply:
np removing brackets, thanks.
Don't wanna rename `oldTX` because variable `tx` is already defined as parameter to the function, its the transaction that _might_ be a double open.